sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxUtils.java

```diff
@@ -78,8 +82,32 @@ public static String normalizePathFull(String filename, String baseDir) {
     return filePath;
   }
 
+  
+  /**
+   * @return returns true for Reactor Projects
+   */
   public static boolean isReactorProject(Project project) {
     return project.isRoot() && !project.getModules().isEmpty();
   }
+  
+  /**
+   * @return returns number of Lines of a File
+   */
+  public static int countLines(String normalPath) {
```

Comment:
There is a line number in the size section of the metrics.
http://docs.sonarqube.org/display/SONAR/Metric+definitions#Metricdefinitions-Size
I am not sure whether the line metric is available when the report is consumed.